docs/src/examples/BottomSheet.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { Pressable, StyleSheet, Text, View } from 'react-native';
+import 'react-native-gesture-handler';
+
+// import { SafeAreaView } from 'react-native-safe-area-context';
+
+import { useSharedValue } from 'react-native-reanimated';
+
+import { PropsWithChildren } from 'react';
+import { TouchableOpacity } from 'react-native';
+import Animated, {
+  SharedValue,
+  useAnimatedStyle,
+  useDerivedValue,
+  withDelay,
+  withTiming,
+} from 'react-native-reanimated';
+
+type BottomSheetProps = PropsWithChildren<{
+  isOpen: SharedValue<boolean>;
+  toggleSheet: () => void;
+  duration?: number;
+}>;
+const BottomSheet = ({
+  isOpen,
+  toggleSheet,
+  duration = 500,
+  children,
+}: BottomSheetProps) => {
+  const height = useSharedValue(0);
+  const progress = useDerivedValue(() =>
+    withTiming(isOpen.value ? 0 : 1, { duration })
+  );
+
+  const sheetStyle = useAnimatedStyle(() => ({
+    ...sheetStyles.sheet,
+    transform: [{ translateY: progress.value * 2 * height.value }],
+  }));
+
+  const backdropStyle = useAnimatedStyle(() => ({
+    ...sheetStyles.backdrop,
```

Comment:
by convention we tend to use `[sheetStyles.backdrop, backdrop]` way of joining styles rather than spreading